spec/factories/rounds.rb

```diff
@@ -27,5 +37,6 @@
     members_can_propose_buckets true
```

Comment:
`members_can_propose_buckets` is whether or not the round *should have* a `proposal` mode, not whether or not it *is in* a `proposal` mode.